pkg/db/overview.go

```diff
@@ -113,7 +113,28 @@ func (h *DBHandler) UpdateOverviewDeployment(ctx context.Context, transaction *s
 	}
 	appInEnv := getEnvironmentApplicationByName(env.Applications, deployment.App)
 	if appInEnv == nil {
-		return fmt.Errorf("could not find application %s in environment %s in overview", deployment.App, deployment.Env)
+		selectApp, err := h.DBSelectApp(ctx, transaction, deployment.App)
+		if err != nil {
+			return fmt.Errorf("could not find application '%s' in apps table, got an error: %w", deployment.App, err)
+		}
+		if selectApp == nil {
+			return fmt.Errorf("could not find application '%s' in apps table: got no result", deployment.App)
+		}
+		env.Applications[deployment.App] = &api.Environment_Application{
+			Version:         0,
+			QueuedVersion:   0,
+			UndeployVersion: false,
```

Comment:
can we set the undeployverison from the DeploymentMetaData as well?

Reply:
It's will be set further down in the code.